pkg/arguments/arguments.go

```diff
@@ -26,14 +26,33 @@ import (
 	"reflect"
 	"strings"
 
-	"github.com/mattn/go-isatty"
+	isatty "github.com/onsi/ginkgo/reporters/stenographer/support/go-isatty"
```

Comment:
@igoihman was this deliberate?
https://github.com/onsi/ginkgo/tree/master/reporters/stenographer/support/go-isatty/
looks like an old, less maintained, copy of upstream https://github.com/mattn/go-isatty/.
Or was this simply replaced by IDE being too clever?

Reply:
=> #201 if accidental.